key/key_test.go

```diff
@@ -0,0 +1,409 @@
+// Copyright (c) 2015 Arista Networks, Inc.  All rights reserved.
+// Arista Networks, Inc. Confidential and Proprietary.
+
+package key_test
```

Comment:
I'm glad I was curious about looking up the note about this `_test` package suffix - thanks :heavy_check_mark: